hazelcast-client-new/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheTestSupport.java

```diff
@@ -64,18 +63,11 @@ public void tearDown() {
     }
 
     protected Config createConfig() {
```

Comment:
the `createConfig()` and `createClientConfig()` methods over overridden in Enterprise Repo tests.